Source/Core/DiscIO/VolumeTri.h

```diff
@@ -0,0 +1,108 @@
+// Copyright 2008 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/Lazy.h"
+#include "DiscIO/Filesystem.h"
+#include "DiscIO/Volume.h"
+#include "DiscIO/VolumeDisc.h"
+
+namespace DiscIO
+{
+class BlobReader;
+enum class BlobType;
+enum class Country;
+class FileSystem;
+enum class Language;
+enum class Region;
+enum class Platform;
+
+class VolumeTri : public VolumeDisc
+{
+public:
+  explicit VolumeTri(std::unique_ptr<BlobReader> reader);
+  ~VolumeTri() override;
```

Comment:
For new code, I recommend implementing the [rule of five](https://en.cppreference.com/w/cpp/language/rule_of_three)